index.html

```diff
@@ -0,0 +1,127 @@
+            </div>
+          </div>
+        
+        
+
+        
+
+        
+    </section>
+    <br>
+    <br>
+    <br>
+    <br>
+    <section id="Services"><center><h1>SERVICES</h1></center>
+
+        <div class="row">
+            <div class="column">
+                <center><h2>INTERNET</h2></center>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dicta ratione repellat corrupti nesciunt. labore quaerat, quam fugiat aperiam dolorum quis voluptate enim voluptatum consequatur!
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Dicta dolor sequi optio necessitatibus quis praesentium, inventore adipisci distinctio obcaecati voluptatum provident sapiente illum earum ullam deserunt deleniti consequuntur reprehenderit cupiditate. Lorem ipsum dolor sit amet, consectetur adipisicing elit. In quo modi accusamus similique vitae sed animi. Hic cumque officia alias cupiditate eos aspernatur suscipit! Necessitatibus error suscipit vel hic repudiandae! Lorem ipsum dolor, sit amet consectetur adipisicing elit. Debitis illo amet eum facere eius, porro sed assumenda quisquam sint reiciendis quae quidem minus. Id temporibus officia unde quaerat praesentium quam. Lorem, ipsum dolor sit amet consectetur adipisicing elit. Pariatur iste architecto impedit, eligendi nisi eveniet, expedita corporis voluptatum nesciunt nulla non reprehenderit ab maxime exercitationem officiis aperiam, quidem porro recusandae?</p>
+            </div>
+            <br>
+            <div class="column">
+                <center><h2>PHONE</h2></center>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dicta ratione repellat corrupti nesciunt. labore quaerat, quam fugiat aperiam dolorum quis voluptate enim voluptatum consequatur!
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Dicta dolor sequi optio necessitatibus quis praesentium, inventore adipisci distinctio obcaecati voluptatum provident sapiente illum earum ullam deserunt deleniti consequuntur reprehenderit cupiditate. Lorem ipsum dolor sit amet, consectetur adipisicing elit. In quo modi accusamus similique vitae sed animi. Hic cumque officia alias cupiditate eos aspernatur suscipit! Necessitatibus error suscipit vel hic repudiandae! Lorem ipsum dolor, sit amet consectetur adipisicing elit. Debitis illo amet eum facere eius, porro sed assumenda quisquam sint reiciendis quae quidem minus. Id temporibus officia unde quaerat praesentium quam. Lorem, ipsum dolor sit amet consectetur adipisicing elit. Pariatur iste architecto impedit, eligendi nisi eveniet, expedita corporis voluptatum nesciunt nulla non reprehenderit ab maxime exercitationem officiis aperiam, quidem porro recusandae?</p>
+            </div>
+            <div class="column">
+                <center><h2>TV</h2></center>
```

Comment:
Please make sure to use justify-content instead of using a center tag

Reply:
The same as before, please make sure you use justify-content instead of using a center tag for better practice.